OpenShift/99_post_install.sh

```diff
@@ -32,6 +33,12 @@ create_bridge(){
   fi
 }
 
+extra_registry(){
```

Comment:
Concerned when we don't need the extra registry.

This seems to set it up that it will always be required.